tools/rockset_migration/rockset_queries.py

```diff
@@ -248,104 +248,102 @@ def backup_lambdas(queries: Dict[str, LambdaQuery], dir: Path) -> None:
 
 # In[ ]:
 
+if __name__ == "__main__":
```

Comment:
idk how this file was originally called, but I want to import it to use some functions but I don't want to run the global code